code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -859,4 +859,18 @@
 	quality = DRINK_GOOD
 	color = "#fffafa"
 	taste_description = "cranberry"
-	glass_name = "glass of sprited cranberry"
\ No newline at end of file
+	glass_name = "glass of sprited cranberry"
+	
+/datum/reagent/consumable/graveyard
+	name = "Graveyard"
+	description = "What Happens when you mix all the sodas in the fountain you get this monstorsity"
```

Comment:
description = "What happens when you mix all the sodas in the fountain? You get this monstrosity!"